arctic/chunkstore/chunkstore.py

```diff
@@ -4,18 +4,16 @@
 import ast
 
 from bson.binary import Binary
-from pandas import Series, DataFrame, concat
+from pandas import concat, DataFrame, Series
 
 from ..store._version_store_utils import checksum
 from ..decorators import mongo_retry
 from .._util import indent
-from ..serialization.pandas_serializer import (serialize, deserialize,
-                                               DataFrameSerializer,
-                                               SeriesSerializer,
-                                               PandasSerializer)
+
+from arctic.serialization.numpy_strings import NumpyString
```

Comment:
The _explicit relative_ imports are good. They allow some amount of code movement / package refactoring after the fact, and I think are now endorsed by Guido as reasonable